src/platform/plugins/shared/controls/public/controls/data_controls/range_slider/get_range_slider_control_factory.tsx

```diff
@@ -243,7 +243,7 @@ export const getRangesliderControlFactory = (): DataControlFactory<
             <RangeSliderControl
               controlPanelClassName={controlPanelClassName}
               fieldFormatter={fieldFormatter}
-              isInvalid={selectionHasNotResults}
+              isInvalid={Boolean(value) && selectionHasNotResults}
```

Comment:
Why the addition of `Boolean(value)` here? Also there's a typo in this line that's been around for a while looks like.

Reply:
I opted to add this check because if the user hasn't selected any values in the range slider, it seems a little confusing to show the "your selection results in no data message" when the preceding controls already return no data.

Also I didn't even notice the typo haha. I'll fix it since I'm in here.